packages/office-ui-fabric-react/src/components/Dialog/DialogContent.styles.ts

```diff
@@ -10,19 +15,134 @@ export const getStyles = (
   const {
     className,
     theme,
+    type,
+    hidden,
+    isMultiline,
   } = props;
 
-  const { palette, semanticColors } = theme;
+  const { palette, fonts } = theme;
+  const dialogLgHeaderBackgroundColor = palette.themePrimary;
 
   return ({
     content: [
-      'ms-DialogContent',
+      type === DialogType.largeHeader && [
```

Comment:
`type === DialogType.largeHeader && 'ms-Dialog-lgHeader'` is a shorter way to do these ones that are just class no style

Reply:
Ah, yeah, I wrote them like that before I copied styles over thinking they'd have styles, thanks.